bool enable
float32 brake_cmd
#not to worry about these rn
bool clear
bool ignore
bool boo_cmd #Brake light On Off 
